javascript/javascript3/week2/promise-visual/main.js

```diff
@@ -0,0 +1,39 @@
+//Visual promise one by one
+// using async/await
+ async function translateOneByOne(){
+    const liMarks = document.querySelectorAll(".marks >li");
+    
+    await moveElement(liMarks[0],{ x: 20, y: 300 } );
+    await moveElement(liMarks[1],{ x: 400, y: 300 } );
+    await moveElement(liMarks[2],{ x: 400, y: 20 } );
+} 
+translateOneByOne() 
+
+//uisng Promise 
+ function translateOneByOne(){
+    const liMarks = document.querySelectorAll(".marks >li");
+    moveElement(liMarks[0],{ x: 20, y: 300 } )
+    .then(() => {return moveElement(liMarks[1],{ x: 400, y: 300 })})
+    .then(() => {return moveElement(liMarks[2],{ x: 400, y: 20 } )})
+    .then(() => {console.log("all are moved one by one")})
```

Comment:
Log something out after each element has been moved..

Reply:
I fixed it. Thanks for your feedback if you think I need to change else please comment :-)